test/Project.toml

```
@@ -21,7 +21,7 @@ CloudMicrophysics = "0.29.0"
 Documenter = "1.15"
 ExplicitImports = "1.13"
 GPUArraysCore = "0.2"
-Oceananigans = "0.101.2"
+Oceananigans = "0.101.2, 0.102"
```

Based on

> The update requires Oceananigans 0.102.1.

I presume this should be more like
```suggestion
Oceananigans = "0.102.1"
```
?